R/scale-.R

```diff
@@ -668,17 +682,17 @@ ScaleContinuous <- ggproto("ScaleContinuous", Scale,
     if (self$is_empty()) {
       return(numeric())
     }
-
+    transformer <- self$transformer
```

Comment:
In addition to soft-deprecating the `trans` field, we may also change these types of statements to `transformer <- self$transformer %||% self$trans`.

Reply:
yeah, that is a good idea.

I'm somewhat disliking `transformer` (I know I brought up that name). as it is so close to `transform`... Maybe something like `transform_obj` or something similar

Reply:
I think `transformation` might also be good. Do you have a strong preference?

Reply:
no that's good as well